frontend/shared/components/src/events/EventOverview.vue

```diff
@@ -256,7 +262,11 @@ defineRoutes([
         paramsToProps: () => {
             return {
                 event: props.event,
-                isNew: false
+                isNew: false,
+                callback: async () => {
+                    props.afterDelete?.();
```

Comment:
Het lijkt me niet zo netjes om afterDelete aan te roepen als je gewoon het event wijzigt, de namen kloppen dan niet. 

Het lijkt me beter om hetzelfde patroon te gebruiken als bij MembesTableView:
```
useGlobalEventListener('members-deleted', async () => {
    tableObjectFetcher.reset(true, true)
})
```

Het blijven doorsturen van callbacks zo diep in een view hierarchie lijkt me anders gewoon te veel onderhoudswerk en veroorzaakt heel harde koppeling. Als de ManageEventsView gewoon luistert naar 'events' van verwijderde Events, kan die zichzelf herladen zonder al te harde dependencies.